testing/botan/APKBUILD

```diff
@@ -0,0 +1,44 @@
+# Contributor:
+# Maintainer:
+pkgname=botan
+_pkgreal=Botan
+pkgver=2.4.0
+pkgrel=0
+pkgdesc="Crypto and TLS for C++11"
+url="https://botan.randombit.net/"
+arch="x86_64 x86"
+license="BSD-2-Clause"
+depends=""
+depends_dev="python boost-dev libressl-dev sqlite-dev bzip2-dev zlib-dev"
```

Comment:
There's no package named `python`, correct name is `python3` or `python2` for Legacy Python. The same for executable.